addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,222 @@ msgctxt "#802"
+
+#. Representation of an epg-based one-time timer.
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#821"
+msgid "One time (guide-based)"
+msgstr ""
+
+#. Representation of a timer schedule
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#822"
+msgid "Repeating"
+msgstr ""
+
+#. Representation of an epg-based timer schedule.
+#: xbmc/pvr/addons/PVRClient.cpp
+msgctxt "#823"
+msgid "Repeating (guide-based)"
+msgstr ""
+
+#empty string with id 824
+
+#. Representation of PVR timer state "disabled"
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#825"
+msgid "Disabled"
+msgstr ""
+
+#. Text for notification that a repeating timer now has state "scheduled" (means enabled)
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#826"
+msgid "Repeating timer enabled"
+msgstr ""
+
+#. Notification text for PVR timer state "disabled"
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#827"
+msgid "Timer disabled"
+msgstr ""
+
+#. Text for notification that a repeating timer has been deleted
+#: xbmc/pvr/timers/PVRTimers.cpp
+msgctxt "#828"
+msgid "Repeating timer deleted"
+msgstr ""
+
+#empty string with id 829
+
+#. label for PVR Timer Weekdays settings dialog
+#: skin.confluence
+msgctxt "#830"
+msgid "Weekdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#831"
+msgid "Mondays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#832"
+msgid "Tuesdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#833"
+msgid "Wednesdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#834"
+msgid "Thursdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#835"
+msgid "Fridays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#836"
+msgid "Saturdays"
+msgstr ""
+
+#. Weekday representation
+#: xbmc/dialogs/GUIDialogPVRTimerWeekdaysSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#837"
+msgid "Sundays"
+msgstr ""
+
+#empty strings from id 838 to 839
+
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+msgctxt "#840"
+msgid "You are about to delete a repeating timer. Do you also want to delete all timers currently scheduled by this timer?"
+msgstr ""
+
+#. Label of a context menu entry to create a new recording timer with advanced configuration options
+#: xbmc/pvr/windows/GUIWindowPVRGuide.cpp
+#: xbmc/pvr/windows/GUIWindowPVRSearch.cpp
+msgctxt "#841"
+msgid "Add advanced timer"
```

Comment:
If I understood your description correctly this context item is automatically opening a timer dialog for a repeating timer, but the type can still be changed. Was just thinking if we should only call it "Add custom timer" which better indicates that the timer can be adjusted? Just an idea, whatever you prefer.